src/helpers/script_helpers.sh

```diff
@@ -83,6 +83,17 @@ function preconditions {
 }
 
 
+# Echos command if condition is true
+function echo_if_true {
+  local git_command="$1"
```

Comment:
Also, terminal_helpers.sh might be a slightly better place for this function, if only because all the other `echo_` functions are there too.